SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -808,6 +820,14 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
                 containerInfoBarMap?.setLabels(primary: lastCustomMarker!.title, secondary: buildingName)
                 changeNavigationButtonVisibility(isVisible: true)
                 displayMarkers(forFloor: floor, isUserNavigating: false)
+            } else {
+                let title = NSLocalizedString("positioning.error.outsideBuilding.title",
+                    bundle: SitumMapsLibrary.bundle,
+                    comment: "Alert title error when the user try to set a custom location outside building")
+                let message = NSLocalizedString("positioning.error.outsideBuilding.message",
+                    bundle: SitumMapsLibrary.bundle,
+                    comment: "Alert title message when the user try to set a custom location outside building")
+                showAlertMessage(title: title, message: message, alertType: .otherAlert)
```

Comment:
Cando o usuario crea un destino personalizado fora do edificio, decidín que sería boa idea mostrar un erro indicando que non se poden crear destinos personalizados. Podes revisar os strings máis abaixos para español e inglés por se necesitan unha volta?

Reply:
Revisados